### ISSUE TITLE
Can not add Tinylog to Maven project because the dependency is a .zip on Maven Central

### ISSUE THREAD

**Issue by nureyev** [tinylog-org/tinylog#55]
It is not possible to add Tinylog 1.2 to a Maven-based project since Maven does not set dependencies to the classpath correctly when they are in .zip format.

Would it be possible to upload a .jar to Maven Central please?

Thanks for the project and your time.

**Comment by nureyev** [comment]
Doing a manual install to the local repository works, but might not be the best solution. 

_Update:_ The following workaround fetches the artifact from Maven Central and installs correctly:

Add the dependency to pom.xml but change `<type>zip</type>` to `<type>jar</type>`
```
<dependency>
    <groupId>org.tinylog</groupId>
    <artifactId>tinylog</artifactId>
    <version>1.2</version>
    <type>jar</type>
</dependency>
```

**Comment by pmwmedia** [comment]
Which IDE and Maven version are you using? I have tried to reproduce the problem with a clean and empty Maven repository. Maven 3.3.3 on Eclipse has automatically fetched and downloaded the tinylog JAR and no ZIP file.

My POM:

```
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>
  <groupId>example</groupId>
  <artifactId>app</artifactId>
  <version>0.0.1-SNAPSHOT</version>
  <dependencies>
  	<dependency>
  		<groupId>org.tinylog</groupId>
  		<artifactId>tinylog</artifactId>
  		<version>1.2</version>
  	</dependency>
  </dependencies>
</project>
```

tinylog files downloaded by Maven:

```
tinylog-1.2.jar
tinylog-1.2.jar.sha1
tinylog-1.2.pom
tinylog-1.2.pom.sha1
```

Did you write `<type>zip</type>`  in your `pom.xml` before? In this case it would be correct that Maven has downloaded a ZIP file as it was requested. Usually I don't define the type, because Maven chooses JAR by default.

**Comment by nureyev** [comment]
Thanks for the reply. 

NetBeans 8.2 (all plugins updated as of today)
Maven 3.5.0
    maven-dependency-plugin:2.6 
    maven-resources-plugin:2.6
    maven-compiler-plugin:3.1

When you right-click on **Dependencies** in the **Project** window and click **Add Dependency...** to add in Tinylog, it is possible NetBeans is filling in an inappropriate `<type>`, which Eclipse does not do.

My workaround was to change the type in the Add dependency dialog from the default of _zip_ to _jar_ after I ran the query to download Tinylog from the Maven Central remote repository.

![add_dependency](https://user-images.githubusercontent.com/18418518/29003424-029c3d4a-7ab7-11e7-8f74-3ac900a1398c.JPG)

**Comment by pmwmedia** [comment]
Thank you! I will analyze the behavior of Netbeans. Maybe I'm going to remove the ZIP archive from Maven Central Repository.

**Comment by pmwmedia** [comment]
I just tested it on my computer with Netbeans 8.2. Fresh download from: https://netbeans.org/downloads/start.html?platform=zip&lang=en&option=javase. ~~On my computer, Netbeans leaves the `type` empty and downloads the tinylog JAR.~~

**Update:** Netbeans uses ZIP type, if you use the search, but the default JAR type, if you use the autocomplete function in the form itself.

**Comment by pmwmedia** [comment]
Solved with release of tinylog 1.3 rc1.
